src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -93,6 +93,13 @@ public function getEntitiesByIds($identifier, array $values)
             $values = array_values(array_filter($values, function ($v) {
                 return (string) $v === (string) (int) $v;
             }));
+        } elseif ('guid' === $metadata->getTypeOfField($identifier)) {
+            $parameterType = Connection::PARAM_INT_ARRAY;
+
+            // Like above, but we just filter out empty strings.
+            $values = array_values(array_filter($values, function ($v) {
+                return (string) $v !== '';
```

Comment:
Anyway, as I already told in PR comment, the purpose here is not validating the uuid. The purpose is just allowing empty values, otherwise an optional uuid is not working.